exercises/practice/hello-world/tests/Template.spec.ts

```diff
@@ -32,7 +32,7 @@ describe('Template', () => {
     });
 
     it('should return hello world', async () => {
-        expect(await template.getHelloWorld()).toStrictEqual(22405534230753928650781647905n);
+        expect(await template.getHelloWorld()).toStrictEqual(5735816763073854918203775149089n);
```

Comment:
Would this also work?

```suggestion
        expect(await template.getHelloWorld()).toStrictEqual("Hello, World!"u);
```

Reply:
No. Not directly!
I've written a function to do that.